explainer/explainer11.md

```diff
@@ -280,8 +308,13 @@ private information (eg keys) rather than depending on security through obscurit
 Second, the WoT Scripting API is designed to not have direct access to
```

Comment:
Where is "first"?
Not sure we should number these paragraphs.

Reply:
I restructured this section to make it clear that the list are only examples, and break out the guidelines as a separate item at the end, and contrast its informative nature with the *normative* security and privacy considerations in each deliverable.